javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,46 @@
+const myCandyBasket = [{ candyType: "sweet", pricePerGram: 0.5 }, { candyType: "toffee", pricePerGram: 1.1 },
```

Comment:
here it would be easier to use one object with type as key and price as value . { sweet :0.5, ..... }